modules/odc-creating-projects-using-developer-perspective.adoc

```diff
@@ -19,12 +19,16 @@ Projects starting with `openshift-` and `kube-` host cluster components that run
 .Procedure
 You can create a project using the *Developer* perspective, as follows:
 
-. In the *Add* view, click the *Project* drop-down menu to see a list of all available projects. Select *Create Project*.
+. Click the *Project* drop-down menu to see a list of all available projects. Select *Create Project*.
 +
 image::odc_create_project.png[Create Project]
 
-. In the *Create Project* dialog box, enter a unique name for the *Name* field. For example, enter `myproject` as the name of the project in the *Name* field.
-. Optional: Add the *Display Name* and *Description* details for the Project.
+. In the *Create Project* dialog box, enter a unique name for the *Name* field. For example, enter `myproject` as the name of the project.
```

Comment:
To shorten, I'd suggest, "...enter a unique name, such as `myproject`, in the **Name** field."